services/ingestion/etl/budg/csv/serverless.yml

```diff
@@ -6,26 +6,39 @@ plugins:
 custom:
   webpackIncludeModules: true
   eubfrEnvironment: ${env:EUBFR_ENV, 'dev'}
+  dynamoTable: ${file(../../../../db/serverless.yml):custom.DYNAMODB_TABLE}
 
 package:
   individually: true
 
 provider:
   name: aws
   runtime: nodejs6.10
+  timeout: 60
   stage: ${opt:stage, file(../../../../../config.json):stage, 'dev'}
   region: ${opt:region, file(../../../../../config.json):region, 'eu-central-1'}
   deploymentBucket:
     name: eubfr-${self:custom.eubfrEnvironment}-deploy
   stackTags:
     ENV: ${self:custom.eubfrEnvironment}
+  iamRoleStatements:
```

Comment:
From [EUBFR-18](https://webgate.ec.europa.eu/CITnet/jira/browse/EUBFR-18) I think this service is to call an SNS and not the DB directly, no?

Reply:
Discussed during our DS. This saves us from facing the concurrent lambdas limit (for now). As we get more confident with the whole AWS infrastructure, we may update it or roll it back, I honestly don't know what's the best solution here